tailwind.config.js

```diff
@@ -0,0 +1,33 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./index.html', './src/**/*.{js,jsx}'],
+  important: true,
+  theme: {
+    extend: {
+      animation: {
+        'slide-down': 'slideDown 300ms ease-out',
+        'slide-up': 'slideUp 300ms ease-out',
+        flip: 'rotate-180 300ms ease-out',
+      },
+      keyframes: {
+        slideDown: {
+          from: {
+            height: 0,
+          },
+          to: {
+            height: 'var(--radix-accordion-content-height)',
+          },
+        },
+        slideUp: {
+          from: {
+            height: 'var(--radix-accordion-content-height)',
+          },
+          to: {
+            height: 0,
+          },
+        },
```

Comment:
**suggestion**: you could use this plugin for animations https://www.npmjs.com/package/tailwindcss-animate

Reply:
I couldn't figure out how to use the special radix vars with this plugin. I'm going to keep this custom animation for now.